website/content/docs/enterprise/license/faq.mdx

```diff
@@ -20,6 +20,58 @@ This check is part of the server boot-up process.
 In previous versions of HashiCorp enterprise products, one server could distribute a license to other servers via the Raft protocol.
 This will no longer work since each server must be able to find a valid license during the startup process.
 
+## Q: What are the upgrade requirements?
+
+All customers on Consul Enterprise 1.8/1.9 must first upgrade their agents to the latest patch release.
+The license file must also be set in an environment variable or file path. Otherwise,
+the Consul agents will fail to retrieve the license with a valid agent token.
+The upgrade process varies if ACLs are enabled or disabled in the cluster. The following table and corresponding sections describe common 1.10.0 upgrade scenarios.
+
```

Comment:
@mkeeler has a whole new section on 1.10.0 upgrades in https://github.com/hashicorp/consul/pull/10446 that go over each scenario in detail. Rather than repeating them here, I think its best to share the most relevant bit of information which is the first paragraph above until "The upgrade process varies if ACLs are enabled or disabled in the cluster", and then link to the new page @mkeeler added to our upgrade section of the docs that's landing in that PR

Reply:
Link to page from #10446 for 1.10.0 will be in `/docs/upgrading/instructions/upgrade-to-1-10-x`, to link to from here

Reply:
Great, we'll remove the table and the subsections and add the link.